pcsx2/gui/Panels/MemoryCardListPanel.cpp

```diff
@@ -519,14 +519,15 @@ void Panels::MemoryCardListPanel_Simple::Apply()
 	for(uint slot=0; slot<8; ++slot) {
 		g_Conf->Mcd[slot].Type = m_Cards[slot].Type;
 		g_Conf->Mcd[slot].Enabled = m_Cards[slot].IsEnabled && m_Cards[slot].IsPresent;
+		wxString toPrint(g_Conf->Mcd[slot].Filename.wstring());
 		if (m_Cards[slot].IsPresent)
 			g_Conf->Mcd[slot].Filename = m_Cards[slot].Filename;
 		else
 			g_Conf->Mcd[slot].Filename = L"";
 
 		if (g_Conf->Mcd[slot].Enabled) {
 			used++;
-			Console.WriteLn(L"slot[%d]='%s'", slot, WX_STR(g_Conf->Mcd[slot].Filename.GetFullName()));
+			Console.WriteLn(L"slot[%d]='%s'", slot, WX_STR(toPrint));
```

Comment:
can `toPrint` just be replaced with `g_Conf->Mcd[slot].Filename.wstring()`?

Reply:
In this case, it has to be explicitly wxString in that overload it won't take wstring.